virtcontainers/qemu_arch_base.go

```diff
@@ -128,7 +128,11 @@ const (
 // 0 is reserved.
 const bridgePCIStartAddr = 2
 
+//TODO: This should be a hypervisor agnostic definition
```

Comment:
Again, please could you resolve, or raise an issue and reference it here.

Reply:
Fixed this by moving it to the common config definitions.